### DISCUSSION TITLE
v2.0.3

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#91] [Announcements]
## Changes

- Don't Query until device rssi > -90dBm
- Query forever w/ exponential backoff (retry after maxes at 1m)
- New name fingerprint for macbooks
- Query: use max power for all connections

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.0.3'>v2.0.3</a>.</em>

**Comment by DTTerastar** [comment]
Okay, fingers crossed I think this get's our reliability up to actually use Query now.  Let me know what you see.

**Reply by martinrosenauer** [reply]
Have 3 nodes that have upgraded now. Hope it's more stable on our iPhones/Apple Watch here. (Experiences phones never expired with v2.0.2 even though they left the premises).

**Reply by DTTerastar** [reply]
I experienced phones that were intermittently gone when they weren't (due to queries failing).  I can't explain what you're seeing.

**Reply by martinrosenauer** [reply]
Thanks Darrell! Very understandable, will keep an eye on it and report back.

Another question - have you experienced any interference on your 2.4ghz Wi-Fi due to ESPresense?

**Reply by DTTerastar** [reply]
BTLE is designed to fit in-between wifi channels 1, 6, and 11.  So as long as you only use those 3 channels (which is what you should do anyways) you shouldn't notice anything.  

If you want to go next level turn off query and active scanning and then you're just "passively" monitoring what is already being sent via devices.

**Comment by DTTerastar** [comment]
I'm seeing this [issue](https://github.com/h2zero/NimBLE-Arduino/issues/302) effect stability now.  Trying to figure out a work around.
